ui/tests/integration/components/keymgmt/key-edit-test.js

```diff
@@ -0,0 +1,76 @@
+import { module, test } from 'qunit';
+import EmberObject from '@ember/object';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | keymgmt/key-edit', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    const now = new Date().toString();
+    let model = EmberObject.create({
+      name: 'Unicorns',
+      id: 'Unicorns',
+      minEnabledVersion: 1,
+      versions: [
+        {
+          id: 1,
+          creation_time: now,
+        },
+        {
+          id: 2,
+          creation_time: now,
+        },
+      ],
+    });
+    this.model = model;
+    this.tab = '';
+  });
+
+  test('it renders show view as default', async function (assert) {
+    // this.set('model', model);
```

Comment:
should this comment be kept or is it a leftover?

Reply:
Right, it's not needed